<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskConsumerKeyAndSecret</class>
 <widget class="QWidget" name="AskConsumerKeyAndSecret">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Consumer key and secret</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textMessage">
     <property name="text">
      <string>Please enter consumer key and secret received from Evernote service for this application</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="consumerKeyEnterBuddy">
     <property name="text">
      <string>Consumer &amp;key:</string>
     </property>
     <property name="buddy">
      <cstring>consumerKeyEnterForm</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="consumerKeyEnterForm"/>
   </item>
   <item>
    <widget class="QLabel" name="consumerSecretEnterBuddy">
     <property name="text">
      <string>Consumer &amp;secret:</string>
     </property>
     <property name="buddy">
      <cstring>consumerSecretEnterForm</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="consumerSecretEnterForm"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string notr="true">Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="shortcut">
        <string notr="true">Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
